custom_components/ha_washdata/suggestion_engine.py

```
@@ -1207,6 +1253,94 @@ def _suggest_min_off_gap(
 
+    #: Device types where the anti-crease/anti-wrinkle tail must be excluded from
+    #: the stop/start min-active statistic (#343).
+    _ANTI_CREASE_DEVICE_TYPES = (
+        DEVICE_TYPE_WASHING_MACHINE,
+        DEVICE_TYPE_DRYER,
+        DEVICE_TYPE_WASHER_DRYER,
+    )
+
+    def _strip_anti_crease_tail(
+        self,
+        ordered_powers: "np.ndarray",
```

Ruff (0.16.1)

[warning] Remove quotes from type annotation

Remove quotes

(UP037)